diff --git a/application/views/pages/integrations.php b/application/views/pages/integrations.php
index 5ac5d5dc..6d51ad37 100755
--- a/application/views/pages/integrations.php
+++ b/application/views/pages/integrations.php
@@ -134,6 +134,29 @@
                     </div>
                 </div>
 
+                <div class="col-sm-6 mb-4">
+                    <div class="card h-100">
+                        <div class="card-header">
+                            <h5 class="fw-light text-black-50 mb-0">
+                                <?= lang('scheducal') ?>
+                            </h5>
+                        </div>
+                        <div class="card-body">
+                            <div class="mb-3 integration-info">
+                                <small>
+                                    <?= lang('scheducal_info') ?>
+                                </small>
+                            </div>
+                        </div>
+                        <div class="card-footer bg-white border-0">
+                            <a href="<?= site_url('scheducal_settings') ?>" class="btn btn-outline-primary w-100">
+                                <i class="fas fa-cogs me-2"></i>
+                                <?= lang('configure') ?>
+                            </a>
+                        </div>
+                    </div>
+                </div>
+
                 <?php slot('after_integration_cards'); ?>
             </div>
         </div>
